src/components/expression/expression.tsx

```diff
@@ -77,11 +76,17 @@ export type EuiExpressionProps = CommonProps & {
    * Sets the display style for the expression. Defaults to `inline`
    */
   display?: keyof typeof displayToClassNameMap;
+  /**
+   * Forces color to display as `danger` and shows an `alert` icon
+   */
   isInvalid?: boolean;
   /**
-   * Sets a custom width for the description when using the columns layout. Defaults to 30%
+   * Sets a custom width for the description when using the columns layout.
+   * Set to a number for a custom width in `px`.
+   * Set to a string for a custom width in custom measurement.
+   * Defaults to `20%`
    */
-  descriptionWidth?: number;
+  descriptionWidth?: number | string;
```

Comment:
This allows the consumer to decide how they want to handle the sizing unit. Whether by pixel or percentage or some other unit like `vw`. Keeps it really flexible by simply passing any type that `style.width` accepts.